webapp/apps/btax/views.py

```diff
@@ -322,6 +324,15 @@ def output_detail(request, pk):
 
     model = url.unique_inputs
     if model.tax_result:
+        exp_comp_dt = url.exp_comp_datetime
+        utc_now = datetime.datetime.utcnow()
+        utc_now = utc_now.replace(tzinfo=pytz.utc)
+        dt = exp_comp_dt - utc_now
+        exp_num_minutes = dt.total_seconds() / 60.
+        exp_num_minutes = round(exp_num_minutes, 2)
+        exp_num_minutes = exp_num_minutes if exp_num_minutes > 0 else 0
+        JsonResponse({'eta': exp_num_minutes, 'wait_interval': 15000}, status=202)
```

Comment:
Great work here @hayleefay. I just have one comment. I don't think we need to re-calculate the wait time here since we know that it will be about 15 seconds. If you want you could remove 327-333 and replace exp_num_minutes with .25.

Sorry I meant to leave this comment a couple days ago.